l10n_ch_payment_slip/payment_slip.py

```diff
@@ -202,8 +202,8 @@ def _compute_scan_line_list(self):
         line = []
         if not self._can_generate(self.move_line_id):
             return []
-        amount = '01%.2f' % self.amount_total
-        justified_amount = amount.replace('.', '').rjust(10, '0')
```

Comment:
We have to round the amount_total to 2 digit to ensure bvr is on 2 digit it precision is changed.